csharp/ql/lib/semmle/code/csharp/dataflow/internal/DataFlowPrivate.qll

```diff
@@ -743,11 +744,11 @@ private module Cached {
       FlowSummaryImpl::Public::SummarizedCallable c,
       FlowSummaryImpl::Private::SummaryNodeState state
     ) {
-      useFlowSummary(c) and
+      c instanceof DataFlowSummarizedCallable and
```

Comment:
You may as well change the type of `c` to be `DataFlowSummarizedCallable`.

Reply:
Oh right!